amlb/runners/aws.py

```diff
@@ -924,6 +924,11 @@ def download_file(obj, dest, dest_display_path=None):
         self._update_instance(instance_id, success=success)
         return success, error
 
+    def _results_summary(self, scoreboard=None):
+        log.info(
```

Comment:
It is technically possible to still provide a result summary, which would be nice. The old version also did not provide a summary either, though. So for now we just have this message (the old PR #452 would simply crash here, because it would format the results of an empty dataframe in the base class level).